package.json

```diff
@@ -44,14 +46,14 @@
   "homepage": "https://github.com/pandell/mocha-testdata",
   "readmeFilename": "README.md",
   "devDependencies": {
-    "chalk": "0.*",
+    "chalk": "^1.1",
     "gulp": "^3.8",
-    "gulp-jshint": "^1.8",
+    "gulp-jshint": "^2.0",
```

Comment:
@milang FYI, In newer versions of `npm` this causing an `unmet peer dependency` error.